WalletWasabi.Fluent/ViewModels/MainViewModel.cs

```diff
@@ -281,7 +281,7 @@ private void RegisterViewModels()
 					{
 						// TODO: Remove reference to WalletRepository when this ViewModel is Decoupled
 						// TODO: Why is this code duplicated?
-						var pwAuthDialog = new PasswordAuthDialogViewModel(walletViewModel.WalletModel, PasswordRequestIntent.Other);
+						var pwAuthDialog = new PasswordAuthDialogViewModel(walletViewModel.WalletModel, "Next");
```

Comment:
Wasn't it `Continue` before?

Reply:
yes it is

Reply:
You're right. I don't know why I changed it...